solidity/package.json

```diff
@@ -14,7 +14,7 @@
 	"dependencies": {
 		"@erc725/erc725.js": "^0.22.0",
```

Comment:
above you bumped erc725 as well. let's bump it to latest here as well @democrazyfi